<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Welcome_window</class>
 <widget class="QWidget" name="Welcome_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notes - Выбор пользователя</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="welocom_message">
         <property name="text">
          <string>Выберите пользователя</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="usernames_box"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="start_button">
         <property name="text">
          <string>Выбрать пользователя</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_user_button">
         <property name="text">
          <string>Удалить пользователя</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="new_user_button">
         <property name="text">
          <string>Создать нового пользователя</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
